docs/tools/tools_intro.md

```diff
@@ -29,8 +29,8 @@ You can build Mbed OS with the Arm Compiler and GUN Arm Embedded toolchains. The
 
 | Compiler| Download location | Name in Mbed CLI |
 | --- | --- | --- |
-| Arm Compiler 6.13 (default ARM toolchain) | - A paid version is available as [Arm Compiler 6.13 Professional](https://developer.arm.com/products/software-development-tools/compilers/arm-compiler/downloads/version-6). </br> - A paid version is also included in [Keil MDK 5.29](http://www2.keil.com/mdk5/529) | `arm` |
-| GNU Arm Embedded version 9 (9-2019-q4-major) | [GNU Arm Embedded version 9 (9-2019-q4-major)](https://developer.arm.com/open-source/gnu-toolchain/gnu-rm/downloads) | `gcc` |
+| Arm Compiler 6.13 (default ARM toolchain) | - A paid version is available as [Arm Compiler 6.13 Professional](https://developer.arm.com/products/software-development-tools/compilers/arm-compiler/downloads/version-6). </br> - A paid version is also included in [Keil MDK 5.29](http://www2.keil.com/mdk5/529) | `ARM` |
```

Comment:
Hang on - are they actually case sensitive?

Reply:
They're not case sensitive, however, Mbed CLI shows the compiler names in uppercase, and the rest of docs and examples also use uppercase.